tensorflow/compiler/tf2tensorrt/utils/trt_allocator.h

```diff
@@ -53,8 +53,8 @@ class TRTDeviceAllocator : public TRTBaseAllocator {
   virtual ~TRTDeviceAllocator() {
     VLOG(1) << "Destroying allocator attached to " << allocator_->Name();
   }
-  void* allocate(uint64_t size, uint64_t alignment, uint32_t flags) override;
-  void free(void* memory) override;
+  void* allocate(uint64_t size, uint64_t alignment, uint32_t flags) noexcept override;
+  void free(void* memory) noexcept override;
```

Comment:
You also add noexcept to the implementation, but I don't see any corresponding change to trt_allocator.cc.

Reply:
The cc files mentioned all have matching noexcept tags now, thanks!